modules/category/category.model.js

```diff
@@ -1,6 +1,4 @@
 const mongoose = require('mongoose');
-const Product = require('../product/product.model');
```

Comment:
Please remove `isMain `and `subcategories `fields from category model.

Reply:
allready remove it